docs/book/basics/simple_resource.md

```diff
@@ -58,8 +58,26 @@ used to configure code generators to run against this code.  The code generators
 generate boilerplate functions and types to complete the Resource definition.
 To learn more on configuring code generation see the *Code Generation* chapter.
 
-Note: The `// +kubebuilder:validation:Pattern=.+:.+` annotation declares Pattern validation
-requiring that the `Image` field match the regular expression `.+:.+`
+**validation**
```

Comment:
`basics` sections are suppose to be the `day0` experience, so I think we should add these topics under `beyond basics` ?

Reply:
Right, make sense.